src/olympia/conf/dev/settings.py

```diff
@@ -17,11 +17,12 @@
 API_THROTTLING = True
 
 DOMAIN = env('DOMAIN', default='addons-dev.allizom.org')
+SERVICES_DOMAIN = env('SERVICES_DOMAIN', default='services.addons-dev.allizom.org')
 SERVER_EMAIL = 'zdev@addons.mozilla.org'
 SITE_URL = 'https://' + DOMAIN
 INTERNAL_SITE_URL = env('INTERNAL_SITE_URL', default='https://addons-dev.allizom.org')
 EXTERNAL_SITE_URL = env('EXTERNAL_SITE_URL', default='https://addons-dev.allizom.org')
-SERVICES_URL = env('SERVICES_URL', default='https://services.addons-dev.allizom.org')
+SERVICES_URL = 'https://' + SERVICES_DOMAIN
```

Comment:
I'm surprised that python doesn't seem to have support for generating URLs including the sechema? Feels wrong to use simple string concat here, especially since it is split across files.

Can you think of a neater solution?

Reply:
IMHO it doesn't matter at all here. Could use `f` strings if you prefer, I went with the same style used in the file already for `SITE_URL`. Using the standard library we could do `urlunsplit(('https', SERVICES_DOMAIN, '', '', ''))` which I don't think is an improvement (or `urlunparse()`, but same problem: we would need to path empty strings for the `path`, `query` and `hash` components of the URL)